client-v2/src/shared/components/Collection.tsx

```diff
@@ -77,26 +89,58 @@ const ItemCountMeta = CollectionMetaBase.extend`
   flex: 0;
 `;
 
-const CollectionHeadingText = styled('span')`
+const CollectionHeadlineWithConfigContainer = styled('div')`
+  flex-grow: 1;
+  max-width: calc(100% - 90px);
+  display: flex;
+`;
+
+const CollectionHeadingText = styled('div')`
+  display: inline-block;
+  max-width: 400px;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  flex: 1;
+`;
+
+const CollectionConfigContainer = styled('div')`
+  display: inline-block;
+  font-family: GHGuardianHeadline-Regular;
+  font-size: 22px;
+  color: #333333;
+  height: 40px;
+  line-height: 40px;
+  white-space: nowrap;
+  margin-left: 3px;
+`;
+
+const CollectionConfigText = styled('div')`
+  display: inline;
+  font-weight: normal;
+  font-style: normal;
+`;
+
+const CollectionConfigTextPipe = styled('span')`
+  color: #c9c9c9;
 `;
 
 const CollectionToggleContainer = styled('div')`
   padding-top: 5px;
   margin-left: auto;
+  z-index: 2;
 `;
 
 const CollectionShortVerticalPinline = ShortVerticalPinline.extend`
   right: initial;
   left: 0;
 `;
 
-class CollectionDetail extends React.Component<Props, { isOpen: boolean }> {
+class CollectionDisplay extends React.Component<Props, { isOpen: boolean }> {
+  public static defaultProps = {
+    isUneditable: false
+  };
   public state = {
-    isOpen: true
+    isOpen: !this.props.isUneditable
```

Comment:
Question: do we ever want a Collection to be opened when it's uneditable?

Reply:
@RichieAHB I think we briefly discussed and came down on the side of yes -- the case being that even if users can't edit collections it's useful for them to see what's in them to make curation decisions -- does this make sense?

Reply:
CP said they wanted the ability to see what was inside a locked collection. A locked Collection is collapsed by default when it loads.

Reply:
👍